modules/33acrossIdSystem.js

```diff
@@ -53,8 +55,15 @@ function calculateQueryStringParams(pid, gdprConsentData) {
     params.us_privacy = uspString;
   }
 
+  if (gppConsent) {
+    const { gppString = '', applicableSections = [] } = gppConsent;
+
+    params.gpp = gppString;
```

Comment:
Does GPP string need to be encoded as well? Wouldn't it contain chars like `=`?

Reply:
hmm according to prebid's example it does https://docs.prebid.org/dev-docs/modules/consentManagementGpp.html#bidder-adapter-gpp-integration . Can we guarantee that all sections will be websafe base 64?